VogtPayroll6/Payroll.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll6
+{
+    class Payroll
+    {
+        public decimal grossPaytotal = 0.00m;
+
+        public void DisplayPayroll(Employee[] empArray) {
+
+            foreach (var employee in empArray)
+            {
+                grossPaytotal += employee.GetGrossPay();
+            }
+
+            Console.WriteLine($"Payroll: {grossPaytotal:C2}");
+            Console.WriteLine();
+      
+        }
+
+        public Employee[] HowManyEmployees()
+        {
+            PayrollConsoleReader payrollConsoleReader = new PayrollConsoleReader();
```

Comment:
dont create a payroll console reader here. its creating a web in terms of responsibilities. your payroll console reader is reading in an employee. display should be pushing an employee to the payroll class